conf/application.conf

```diff
@@ -157,6 +157,7 @@ auth {
   enrolments {
     team = "classification"
     manager = "classification-manager"
+    read-only = "classification-read-only"
```

Comment:
I believe @SteveSugden and @pllaurado have implemented using this in https://github.com/hmrc/tariff-classification-frontend/pull/190